csrc/scheduler/matmul.cpp

```diff
@@ -437,6 +436,186 @@ void scheduleProlog(TensorView* shared_mem_tv, const MatmulParams& params) {
           .propagateParallelType());
 }
 
+//! swizzle the shared mem data layout using a same method in prologSwizzle.
+//! The shift parameter is added for the transform of MMA results to skip the
+//! K axis and will skip the actual swizzle. This is to ensure a same transform
+//! history between the MMA result tensor and the epilogue shared memory tensor
+//! so the corresponding domains of these two tensors can be mapped. This
+//! function may be merged with prologSwizzle as they are using a same method.
+int epilogSwizzle(
```

Comment:
Could you don't copy-paste code, but instead unify prologue swizzle and epilogue swizzle? Maybe write a function like `swizzleSharedMemory`, and call it both in prologue and epilogue?

Reply:
I am actually waiting for this request. done.